lib/plug/cowboy/handler.ex

```diff
@@ -4,6 +4,13 @@ defmodule Plug.Cowboy.Handler do
   @already_sent {:plug_conn, :sent}
 
   def init(req, {plug, opts}) do
+    start = System.monotonic_time()
+
+    :telemetry.execute([:plug_adapter, :request, :start], %{
```

Comment:
tbh, I don't really understand the event name. I don't see the concept `:request` really in this library. Plug has an adapter concept, but nothing is explicitly called `plug_adapter`. Maybe `[:plug, :adapter, :handler, :start]`?

If I had my way, I'd go with `[:plug_cowboy, :handler, :start]` since that's a pretty literal mapping of the code that's being executed.

Reply:
The question is if a `handler` is a requirement of `plug_adapter` or just an implementation naming thing here in this library. It would just need to be consistent across adapters. Does anyone know the details?

Reply:
🤷 As long as it's a static name, I'll live with it